mypy/test/testcheck.py

```diff
@@ -163,21 +176,20 @@ def run_case_once(
             a = normalize_error_messages(a)
 
         # Make sure error messages match
-        if incremental_step == 0:
+        if incremental_step < 2:
             # Not incremental
```

Comment:
This comment seems outdated

Reply:
66d8305e2db69e24fc9566aea13e39c514ce1b20